dbSearcher/header.php

```diff
@@ -6,20 +6,23 @@
         <link rel="stylesheet" href="css/styles.css" />
         <script>
             function copy(QueryId){
-                // Get all the querys made.
-                let querys = document.querySelectorAll("#query");
-
-                // Extract ours.
-                let temp = querys[QueryId].innerHTML;
+                // Get text of clicked element
```

Comment:
consistency, end comments with a dot if sentence

Reply:
of course.